docs/dev/specs/packable.md

```diff
@@ -324,32 +324,32 @@ There is only one optional feature for this crate:
 One disadvantage of using `PackError` and `UnpackError` as the error variant
 returned by the `Packable::pack` and `Packable::unpack` methods is that
 conversions between errors must be explicit.  As seen in the `Maybe` example,
-we use the `UnpackError::coerce` method to map the error variant to an
-appropiate type. The signature of this method is
+we use the `.coerce()` method to map the error variant to an appropiate type.
+These methods are provided by the following traits:
```

Comment:
Just curious, you use plural because of the generics ?

Reply:
No, I use plural because I wrote something different before and forgot to update this part :trollface: